packages/core/src/resend/audience.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Minimal helpers for working with Resend Audience contacts without importing
+ * the Resend SDK. We model only the parts of the response we read so callers
+ * can pass the real `resend.contacts` client directly and get strong typing.
+ */
+
+export interface ResendGetContactResponseLike {
+  readonly data: { readonly unsubscribed?: boolean } | null;
+  readonly error?: unknown;
+}
+
+export interface ResendContactsClientLike {
```

Comment:
@CharlieHelps please get rid of these handwritten types and just use the native Resend SDK (there is nothing wrong with importing it here!)